vayesta/core/types/wf/fci.py

```diff
@@ -479,9 +506,42 @@ def as_cisdtq(self, c0=None):
         assert(c4_aabb_pack.shape == (ij_pairs_a * ab_pairs_a, ij_pairs_b * ab_pairs_b))
         # abbb
         c4_abbb_pack = np.einsum('i,j,ij->ij', t1signa, t3signb, self.ci[t1addra[:,None], t3addrb])
-        assert(c4_aabb_pack.shape == (nocca * nvirb, ijk_pairs_b * abc_pairs_b))
+        assert(c4_abbb_pack.shape == (nocca * nvirb, ijk_pairs_b * abc_pairs_b))
 
         # Now, unpack... TODO
+        c4_aaaa = decompress_axes(
+                "iiiiaaaa",
+                c4_aaaa_pack,
+                shape=(nocca, nocca, nocca, nocca, nvira, nvira, nvira, nvira),
+                symmetry="--------",
+        )
+        c4_bbbb = decompress_axes(
+                "iiiiaaaa",
+                c4_bbbb_pack,
+                shape=(noccb, noccb, noccb, noccb, nvirb, nvirb, nvirb, nvirb),
+                symmetry="--------",
+        )
+        c4_aaab = decompress_axes(
+                "iiiaaajb",
+                c4_aaab_pack,
+                shape=(nocca, nocca, nocca, nvira, nvira, nvira, noccb, nvirb),
+                symmetry="--------",
+        )
+        c4_aaab = c4_aaab.transpose(0, 1, 2, 6, 3, 4, 5, 7)
+        c4_aabb = decompress_axes(
+                "iiaajjbb",
+                c4_aabb_pack,
+                shape=(nocca, nocca, nvira, nvira, noccb, noccb, nvirb, nvirb),
+                symmetry="--------",
+        )
+        c4_aabb = c4_aabb.transpose(0, 1, 4, 5, 2, 3, 6, 7)
+        c4_abbb = decompress_axes(
+                "iajjjbbb",
+                c4_abbb_pack,
+                shape=(nocca, nvirb, noccb, noccb, noccb, nvirb, nvirb, nvirb),
+                symmetry="--------",
+        )
+        c4_abbb = c4_abbb.transpose(0, 2, 3, 4, 1, 5, 6, 7)
```

Comment:
Hi Ollie, can you please try to stick to the existing way of wrapping lines in Vayesta (fill each line unless you go beyond 120), instead of putting one argument per line - thanks

Reply:
All fit in 120 now